config/README.md

```diff
@@ -38,33 +38,35 @@ configuration from many of the services.
 
 ## Building a Test Network (TestNet)
 
-`config-builder` builds an entire network's or swarm's configuration including the
-genesis blob. It takes as one of its input parameters an index that selects, which
-of these configs to return. This can be used to create a Libra TestNet by specifying
-distinct indices for each validator. Similarly the tool can be used to add Fullnodes
-to an existing network.  Finally, it enables generation of a mint/faucet client
-capable of performing mint transactions/creating accounts.
+`config-builder` builds an entire configuration for a Validator or FullNode, 
+including the genesis blob. It takes as one of its input parameters an index that
+specifies the specific node config to return. This can be used to create a Libra 
+TestNet by constructing compatible configurations the full set of Validators.
+Similarly the tool can be used to add Fullnodes to an existing network.  Finally, 
+it enables generation of a mint/faucet client capable of performing mint 
+transactions/creating accounts.
 
 ## Generating a new TestNet
 
-The only requirements for generating a TestNet configuration are: (i) having IP
-addresses and ports for each Libra node; (ii) a pre-agreed upon shared secret;
-and (iii) a fixed ordering for each Node in the network. Full node networks can
-either be added to existing configs or generate completely new configs.
+The only requirements for generating the configuration for a full TestNet are: (i) 
+having IP addresses and ports for each Validator; (ii) a pre-agreed upon shared secret
+for the Validator network; and (iii) a fixed ordering for each Validators in the 
+network. FullNode configs can either be added to existing configs or generated from 
+scratch. 
```

Comment:
minor nit, would avoid potential colloquialisms like "from scratch" and intead

configs can be generated or added to existing configs